docs/source/basics/customizing_interface.md

```diff
@@ -51,473 +51,84 @@ def foo(
     - item2
     - item3
 
-    ## headers
+    ### headers
 
-    paragraphs
-    
-    The line starting in `__metadata__:` signals the end of the long workflow description
-
-    __metadata__:
-        ...
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
+    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo 
+    consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse 
+    cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat 
+    non proident, sunt in culpa qui officia deserunt mollit anim id est 
+    laborum.
     """
     ...
 ```
 
-## Sidebar Customization
-
-After the long workflow description, you have the opportunity to specify
-standard workflow metadata that will get rendered in the sidebar. The currently
-supported metadata fields are `display_name`, `documentation`, `author` (`name`, `email`,
-`github`), `repository`, and `license`. The only constrained field is `license`, which
-should be an identifier from [spdx](https://spdx.org/licenses/).
-
-```yaml
-__metadata__:
-    display_name: My Workflow
-    documentation: https://github.com/author/my_workflow/README.md
-    author:
-        name: Workflow Author
-        email: licensing@company.com
-        github: https://github.com/author
-    repository: https://github.com/author/my_workflow
-    license:
-        id: MIT
-```
-
-## Customizing Parameter Presentation
+### The `LatchMetadata` Object
 
-When a workflow is registered, a each parameter will receive a frontend
-component to ingest values in the browser. These components will perform
-HTML-native type validation on inputted values and can be customized from
-the python code.
-
-The LatchBio Console currently supports customized parameter presentation in the
-following ways:
-
-* display names
-* sections
-* hidden parameters
-* custom component presentations (eg. line vs paragraph presentation of `str`)
-
-Developers can modify this custom presentation by writing statements in
-[DSL](https://en.wikipedia.org/wiki/Domain-specific_language), either in YAML
-format in the workflow function docstring or as JSON format specified in line
-with Python's `typing.Annotated`.
-
-Reading this documentation and browsing examples is the best way to become
-familiar with this DSL.
-
-### Display Name
-
-This must be filled out for each parameter or the parameter will not render in
-the interface. _This option must be specified in the docstring only._
+The main object that organizes the metadata for a workflow is the `LatchMetadata` object. To use, create a singleton instance of a `LatchMetadata` object as follows:
 
 ```python
-@workflow
-def test(
```

Comment:
Is there a way to make our docs versioned? 

It's still useful to have the old docstring docs for the users who used that version. But for new users going forward, it's highly recommended they use the new Python objects.